src/FluentMigrator.Runner.Firebird/Generators/Firebird/FirebirdTypeMap.cs

```diff
@@ -22,20 +22,84 @@
 
 namespace FluentMigrator.Runner.Generators.Firebird
 {
+    /// <summary>
+    /// Represents a specialized type map for the Firebird database.
+    /// </summary>
+    /// <remarks>
+    /// This class provides mappings between .NET <see cref="DbType"/> values and their corresponding Firebird SQL types.
+    /// It extends the <see cref="TypeMapBase"/> class and implements the <see cref="IFirebirdTypeMap"/> interface.
```

Comment:
This kind of comments make me think of this 😅

![Image](https://github.com/user-attachments/assets/cd6da43d-1f92-420b-97cc-72117542d2a5)

Do we need to repeat the implements and inherits ?

Reply:
Yeah, that's great that you do this ! About warnings, did you see my suggestion about [IQuoter warnings](https://github.com/fluentmigrator/fluentmigrator/issues/1920#issuecomment-3204484065) ? I tried it and it works well